neo/Helper.cs

```diff
@@ -181,6 +172,11 @@ public static uint ToTimestamp(this DateTime time)
             return (uint)(time.ToUniversalTime() - unixEpoch).TotalSeconds;
```

Comment:
is this used? @vncoelho ?

Reply:
I kept network in seconds, such as ping, for exmaple, or version payload.